<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VatWindow</class>
 <widget class="QMainWindow" name="VatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 226, 232);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>20</y>
      <width>401</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>13</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #12222D;
font-family: 'RF Dewi';
font-style: normal;
font-weight: 700;
font-size: 16px;
line-height: 120%;
text-align: center;</string>
     </property>
     <property name="text">
      <string>УРОВЕНЬ ФОТОПОЛИМЕРА</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>686</x>
      <y>30</y>
      <width>94</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/back.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>85</y>
      <width>251</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/vat.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>353</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/vat_alarm.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="loadLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>275</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: 'RF Dewi';
font-style: normal;
font-weight: 900;
font-size: 26px;
color: #D83A56;
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="abountLabel">
    <property name="geometry">
     <rect>
      <x>537</x>
      <y>278</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/vat_amount_10.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>261</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/vat_w_amount.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="inLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>105</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/vat_in.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="outLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>105</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/vat_out.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>541</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/vat_work.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="inButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="outButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border : 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="connectionWifi">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/img/noSignal.png</pixmap>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>label_2</zorder>
   <zorder>label_16</zorder>
   <zorder>backButton</zorder>
   <zorder>label_3</zorder>
   <zorder>label_12</zorder>
   <zorder>label_10</zorder>
   <zorder>label_6</zorder>
   <zorder>outLabel</zorder>
   <zorder>abountLabel</zorder>
   <zorder>loadLabel</zorder>
   <zorder>outButton</zorder>
   <zorder>inLabel</zorder>
   <zorder>inButton</zorder>
   <zorder>connectionWifi</zorder>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
